app/Sanatizer.php

```diff
@@ -8,11 +8,133 @@
 
 class Sanatizer extends Model
 {
-    
+    public $folderToSanatize;    
 
-    public function __construct()
+    public function __construct($folderToSanatize, $cleanedSurasPath)
     {
+        $this->folderToSanatize = $folderToSanatize;
+        $this->cleanedSurasPath = $cleanedSurasPath;
+    }
+
+    public function sanatize()
+    {
+        $rawHTMLFiles = scandir($this->folderToSanatize);
+        foreach ($rawHTMLFiles as $rawHTMLFile) {
+            if (($rawHTMLFile != '.')&&($rawHTMLFile != '..')) {
+                $suraStringToClean = self::getSuraString($rawHTMLFile);
+                $readyToSaveSura = self::cleanSuraString($suraStringToClean);
+
+                $suraNumber = substr($rawHTMLFile, 0, strpos($rawHTMLFile, "."));
+                $fileNameToSave = $suraNumber.$readyToSaveSura["suraName"];
+                self::saveSura($readyToSaveSura["theSura"], $fileNameToSave);
+            }
+        }
+    }
+    public function getSuraString($rawHTMLFile)
```

Comment:
this method and the one under it should be under the `private` scope, 
```php
private function getSuraString($rawHTMLFile)
```

references: 
- https://stackoverflow.com/questions/1020749/what-are-public-private-and-protected-in-object-oriented-programming
- https://stackoverflow.com/questions/1020749/what-are-public-private-and-protected-in-object-oriented-programming